memory/migrate_pages.py.data/node_move_pages.c

```diff
@@ -328,17 +330,26 @@ int main(int argc, char *argv[])
 			chunks = strtoul(optarg, NULL, 10);
                         break;
                 case 'o':
+#ifdef HAVE_HUGETLB_HEADER
 			overcommit = 1;
-                        break;
-                case 't':
+#else			
+                        overcommit = 0;
```

Comment:
No need to set the value to 0. Default value is already set to 0. Drop the else.